lib/transformer.js

```diff
@@ -131,7 +131,7 @@ function createClassPrefix(classPrefix) {
 }
 
 function createIdPrefix(idPrefix) {
-    var url_pattern = /^url\(#.+\)$/i;
+    var url_pattern = /url\(#.+?\)/gi;
```

Comment:
it wont work for this 

`<path mask="url(#foo")>`

I guess keeping both can be better

Reply:
@anikethsaha why wouldn't it work? 
It passes the tests though.